Packs/Infoblox/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+
+##### Infoblox
+
+- Added the *network*, *from_ip*, *to_ip*, *status* arguments to the ***infoblox-get-ip*** command.
```

Comment:
@kgal-pan Let's mention that we have fixed/changed the context outputs for this command as well.

Reply:
Added in https://github.com/demisto/content/pull/32619/commits/826b2eaccf88175d6aa19635ddad3d6709ae6118.